src/agoradatatools/etl/load.py

```diff
@@ -160,3 +162,20 @@ def dict_to_json(df: dict, staging_path: str, filename: str) -> str:
     json.dump(df_as_dict, temp_json, cls=NumpyEncoder, indent=2)
     temp_json.close()
     return temp_json.name
+
+
+def list_to_json(df: List[Dict[str, Any]], staging_path: str, filename: str) -> str:
+    """Converts a list into a JSON file.
+
+    Args:
+        df (list): List to be converted to a JSON file
+        staging_path (str): Path to staging directory
+        filename (str): name of JSON file to be created
+
+    Returns:
+        str: Returns a string containing the name of the new JSON file
+    """
```

Comment:
Looking at the `dict_to_json` function, it calls `remove_non_values` to clean up the dict before writing to JSON. I think this needs to be done to all the dicts in the list in this function. But... here's a question. Instead of outputting your transform as a `list`, what if you made it a `dict` where each item was another `dict` instead? This would bypass needing to add the `list_to_json` function + unit tests for it. I'm not sure if the JSON output will be identical but it might be worth testing?

Reply:
This conversation has come up in a number of places, just bringing all of my thoughts to this thread. 

Yesterday I [said](https://github.com/Sage-Bionetworks/agora-data-tools/pull/148#discussion_r1781720733) that it would be ideal if the data structure produced was one that we already support (dict or df) if feasible but I was [fine](https://github.com/Sage-Bionetworks/agora-data-tools/pull/148#discussion_r1781864436) with adding the ability to handle lists if that made the most sense.

Reply:
Lets do it! I'll output it as a `dict` to make it easier for all of us to maintain the repo going forward (and aboid extra unit tests haha) :)